Ромахина Галина

GalinaRoma opened the pull request:
[Посмотреть решение](https://rawgit.com/GalinaRoma/markup-task-2/master/tasks/1.html)

GalinaRoma committed:
Моё решение задачи "Ацссорти"

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

GalinaRoma committed:
Убрала пробел после ; в 16.html

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bratva commented on tasks/12.html:
```diff
@@ -11,7 +11,17 @@ <h1 class="global-heading">&nbsp;</h1>
+                    background: #ffb366;
+                }
+
+                div:nth-of-type(2n),
```
Можно лучше. Можно выделить только пустые элементы. Попробуй переписать

bratva commented on tasks/13.html:
```diff
@@ -11,7 +11,11 @@ <h1 class="global-heading">&nbsp;</h1>
         <main class="global-results">
             <style>
-                /* your styles here */
+                div:nth-of-type(4n+1) > div 
```
Тоже можно лучше. Обрати внимание, что покрасить нужно только определенный тег

bratva commented on tasks/19.html:
```diff
@@ -11,7 +11,36 @@ <h1 class="global-heading">&nbsp;</h1>
+                    background: #fff;
+                }
+
+                div:first-child > div:last-child
```
Лишнее использование комбинатора `>`.

bratva commented on tasks/20.html:
```diff
@@ -11,7 +11,18 @@ <h1 class="global-heading">&nbsp;</h1>
             <style>
-                /* your styles here */
+                div div:first-child:nth-last-child(3),
+                div div:first-child:nth-last-child(3) ~ div
```
зачем селектор первого элемента? А, если будет 5 дочерних элементов, то ты напишешь `div div:first-child:nth-last-child(5)`? Можно универсальнее, попробуй переписать

bratva commented on tasks/9.html:
```diff
@@ -11,7 +11,31 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
-                /* your styles here */
```
Тут не работает ховер по посещенной ссылке, нужно исправить

bratva reviewed (commented):
Привет.  Есть замечания, нужно исправить 🍅

GalinaRoma committed:
В 9 увеличила специфичность селектора для hover и active, приписав туда название класса.
В 12 поняла, что там были пустые и непустые div. Поправила селекторы.
В 13 поправила селектор с тем, что нам нужен определенный тег.
В 19 убрала лишний комбинатор.
В 20 упростила селектор и сделала его более универсальным.

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты
